cl/favorites/tests.py

```diff
@@ -333,6 +334,37 @@ def test_user_can_change_favorites(self) -> None:
         self.assert_text_in_node("Renamed Favorite", "body")
         self.assert_text_in_node("Modified Notes", "body")
 
+    def test_revert_usertag(self) -> None:
+        # Can we revert an object that is being tracked with django-pghistory?
+        tag_name = "test-tag"
+        params = {"username": "kramirez"}
+        test_user = UserFactory.create(
+            username=params["username"],
+            email="test@courtlistener.com",
+        )
+
+        # Object is created with view_count = 0, event object created
+        test_tag = UserTag.objects.create(
+            user=test_user, name=tag_name, title="Test tag"
+        )
+
+        path = reverse(
+            "view_tag",
+            kwargs={"username": params["username"], "tag_name": tag_name},
+        )
+        request = RequestFactory().get(path)
+
+        # Increment view counter by 1, event object created
+        increment_view_count(test_tag, request)
+        self.assertEqual(test_tag.view_count, 1)
+
+        # Revert object to previous change, we use the second-to-last because
+        # the latest event always contains the current version of the model
+        test_tag = test_tag.event.order_by("-pgh_id")[1].revert()
```

Comment:
Maybe I'm wrong, but this approach doesn't seem totally trustworthy to me. From my eyes, you're running the revert command on the `event` model manager, which is returning a model instance through some magic. I'm not _completely_ convinced that the model instance isn't coming from the `event` table. 

I think I'd rather see something simpler, like this:

```suggestion
        # Check that view count set to 0 again
        test_tag = test_tag.event.order_by("-pgh_id")[1].revert()
        self.assertEqual(test_tag.view_count, 0)
  
        # Grab the original item from the DB, just to be sure
        test_tag.refresh_from_db()
        self.assertEqual(test_tag.view_count, 0)
```

Reply:
Inspecting the revert() property inside the library I saw that it performs an update_or_create using the pk and the values from event object as default and then returns the only object that is supposed to be updated.
(https://github.com/Opus10/django-pghistory/blob/001bbfeb5ee7931d6e9b72df965a9ae69d644502/pghistory/models.py#L175)

I was thinking to use a get to get the updated object, but i prefer your suggestion